src/applications/check-in/package.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name" : "@department-of-veterans-affairs/applications-check-in",
+    "version": "1.0.0",
+    "scripts": {
+        "demo":"echo DEMOING $npm_package_name v $npm_package_version with WORKSPACES"
+      },
+    "licence": "MIT",
+    "dependencies":{
+        "@sentry/browser": "^6.13.2",
+        "date-fns": "^2.24.0",
+        "date-fns-tz": "^1.1.1",
+        "i18next": "^21.6.14",
+        "i18next-browser-languagedetector": "^6.1.3",
+        "i18next-resources-to-backend": "^1.0.0",
+        "prop-types": "^15.6.1",
+        "react-i18next": "^11.15.6",
+        "reselect": "^2.5.4",
+        "url-search-params": "^1.1.0"
+    },
+    "devDependencies":{
+        "@testing-library/dom": "^7.26.6",
+        "@testing-library/react": "^11.1.2",
+        "chai": "^4.3.4",
+        "mocker-api": "^2.9.0",
+        "mockdate": "^3.0.5",
+        "sinon": "^3.2.1"
+    },
+    "peerDependencies":{
+        "react": "*",
+        "react-redux": "*",
+        "redux-mock-store": "*",
+        "react-router": "*",
+        "redux": "*",
+        "lodash": "*",
```

Comment:
It would be nice if we could avoid loading lodash since we don't use it anywhere in check-in. Thoughts @acrollet & @wullaski

Reply:
we decided to address this in a follow-on issue in slack. FWIW tons of applications are using lodash so we'll probably have to wait for all apps to be in workspaces for it to take any effect.

Reply:
https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/47496